### PR TITLE
Create TEMPLATE_showcase.md

### LINKED ISSUE #64: Create a TEMPLATE_showcase.md
There is currently no template for a Showcase page.
These pages are similar to a Tool Assembly page in RDMkit.

### PR DESCRIPTION
**Pull request by rabuono**
Closes #64 
Originally copied from RDMkit's TEMPLATE_tool_assembly.md Modified to reflect IDTk nomenclature

### CODE REVIEW

File: showcase/TEMPLATE_showcase.md [hunk 1]
```diff
@@ -0,0 +1,40 @@
...
+<!--- Showcase pages should detail a particular combination of standards and tools from an infrastructural or domain perspective to tackle infectious diseases related data challenges. --->
+
+## Introduction 
+
+<!--- In this section you should provide a brief overview of the context that makes Showcase necessary --->
+
+
+## What is the SHOWCASE?
+
+<!--- In this section you should provide a ... --->
```
Comment (LianeHughes) [inline showcase/TEMPLATE_showcase.md:28]:
```suggestion
<!--- In this section you should provide a brief description of what the showcase is i.e. what it comprises of and a general description for it.  --->
```

File: showcase/TEMPLATE_showcase.md [hunk 2]
```diff
@@ -0,0 +1,40 @@
...
+
+## Who is the SHOWCASE intended for?
+
+<!--- In this section you should provide a ... --->
```
Comment (LianeHughes) [inline showcase/TEMPLATE_showcase.md:32]:
```suggestion
<!--- In this section you should provide a brief account of the target audience or intended users for the showcase --->
```

File: showcase/TEMPLATE_showcase.md [hunk 3]
```diff
@@ -0,0 +1,40 @@
...
+
+## What can you use the SHOWCASE for?
+ 
+ <!--- In this section you should provide a ... --->
```
Comment (LianeHughes) [inline showcase/TEMPLATE_showcase.md:36]:
```suggestion
 <!--- In this section you should provide a brief summary of the uses of the showcase, i.e. when you would use this showcase resource --->
```

**Review by LianeHughes — COMMENTED** [review]
I have quickly added some sentences that we can use as a basis for this template. Please feel free to add more or make changes.

**Comment by rabuono** [comment]
We should add a section for a graphical representation of the Showcase

File: showcase/TEMPLATE_showcase.md [hunk 4]
```diff
@@ -0,0 +1,37 @@
...
+  - name:
+    registry:
+    url:
+---
+
+<!--- Showcase pages should detail a particular combination of standards and tools from an infrastructural or domain perspective to tackle infectious diseases related data challenges. --->
+
+## Introduction 
+
+<!--- In this section you should provide a brief overview of the context that makes Showcase necessary --->
```
Comment (rabuono) [inline showcase/TEMPLATE_showcase.md:23]:
```suggestion
<!--- In this section you should provide a brief overview of the context that makes Showcase necessary. It is useful to mention the projects under which the showcase was created, the involved research infrastructures, and the disease it is meant to tackle --->
```

**Pull request merged**